src/Microsoft.AspNetCore.WebHooks.Receivers/Middleware/WebHookReceiverMiddleware.cs

```diff
@@ -57,12 +57,14 @@ public static void ConfigureMapping(IApplicationBuilder app)
         /// <returns></returns>
         private static async Task ProcessWebHook(HttpContext context, string webHookReceiver, string id)
```

Comment:
I would change this into a proper middleware class so that dependencies can be injected once instead of being pulled out of the container manually.